### ISSUE TITLE
False positive on self.

### ISSUE THREAD

**Issue by chiliec** [erica/testlint#2]
``` swift
endpoint.update().onComplete { _ in
    let emergency = EmergencyRoomsDataSource(database: self.database)
}
```

In closure, testlint say I don't need to use self.

**Comment by erica** [comment]
- For now, annotate with // nwm to skip warning
- Once Swift goes open source, I may be able to do better testing for closures

-- E
